mirascope/chat/utils.py

```diff
@@ -12,26 +12,45 @@
 from .tools import OpenAITool, openai_tool_fn
 
 
-def _get_openai_chat_messages(
-    prompt: Prompt,
-) -> list[ChatCompletionMessageParam]:
-    """Returns a list of messages parsed from the prompt."""
-    return [
-        cast(ChatCompletionMessageParam, {"role": role, "content": content})
-        for role, content in prompt.messages
-    ]
-
-
 def get_openai_messages_from_prompt(
     prompt: Union[Prompt, str],
 ) -> list[ChatCompletionMessageParam]:
     """Returns a list of messages parsed from the prompt."""
     if isinstance(prompt, Prompt):
-        return _get_openai_chat_messages(prompt)
+        return [
+            cast(ChatCompletionMessageParam, {"role": role, "content": content})
+            for role, content in prompt.messages
+        ]
     else:
         return [cast(ChatCompletionMessageParam, {"role": "user", "content": prompt})]
 
 
+def setup_openai_kwargs(
```

Comment:
not sure if there is a protocol for this but, perhaps patch_openai_kwargs or something to that effect informing us that you are mutating the dictionary.

Reply:
Done.